File: Final_MAD/service_java/app/src/main/java/com/example/service_java/MainActivity.java
package com.example.service_java;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {

        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

    }
    public void startService(View view){

            startService(new Intent(this,MusicService.class));

    }
    public void stopService(View view){

            stopService(new Intent(this,MusicService.class));

    }
}

File: HelloWorld/app/src/main/java/com/example/helloworld/MainActivity.kt
package com.example.helloworld

import androidx.appcompat.app.AppCompatActivity //Ensure Compatible on different Devices
import android.os.Bundle
import android.widget.Button
import android.widget.TextView
import android.widget.Toast
import org.w3c.dom.Text

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        val btnclickme = findViewById(R.id.button) as Button
        val myTextView=findViewById(R.id.textView) as TextView
        var timeClicked=0
        // set on-click listener
        btnclickme.setOnClickListener {
            timeClicked=timeClicked+1
            myTextView.text=timeClicked.toString()
            Toast.makeText(this@MainActivity, "Hi Jawad.", Toast.LENGTH_SHORT).show()
        }
    }
}